File: Roboti/app/src/test/java/cz/muni/fi/umimecesky/utils/UtilTest.java
package cz.muni.fi.umimecesky.utils;

import org.junit.Test;

import static org.hamcrest.CoreMatchers.is;
import static org.junit.Assert.assertThat;

public class UtilTest {

    @Test
    public void probabilityTrue_AlwaysTrue() throws Exception {
        assertThat(Util.probabilityTrue(1.0), is(true));
    }

    @Test
    public void probabilityTrue_AlwaysFalse() throws Exception {
        assertThat(Util.probabilityTrue(0.0), is(false));
    }

    @Test
    public void roundBy2places_roundUp() throws Exception {
        assertThat(Util.roundBy2places(12.3456), is(12.35));
    }

    @Test
    public void roundBy2places_roundDown() throws Exception {
        assertThat(Util.roundBy2places(12.3446), is(12.34));
    }
}

File: Roboti/app/src/test/java/cz/muni/fi/umimecesky/utils/UtilTest.kt
package cz.muni.fi.umimecesky.utils

import org.hamcrest.CoreMatchers.`is`
import org.junit.Assert.assertThat
import org.junit.Test

class UtilTest {

    @Test
    @Throws(Exception::class)
    fun probabilityTrueIsAlwaysTrue() {
        assertThat(Util.probabilityTrue(1.0), `is`(true))
    }

    @Test
    @Throws(Exception::class)
    fun probabilityTrueIsAlwaysFalse() {
        assertThat(Util.probabilityTrue(0.0), `is`(false))
    }

    @Test
    @Throws(Exception::class)
    fun roundBy2placesShouldRoundUp() {
        assertThat(Util.roundBy2places(12.3456), `is`(12.35))
    }

    @Test
    @Throws(Exception::class)
    fun roundBy2placesShouldRoundDown() {
        assertThat(Util.roundBy2places(12.3446), `is`(12.34))
    }
}